server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestResetFeatureStateAction.java

```diff
@@ -43,8 +44,7 @@ public String getName() {
 
     @Override
     protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient client) throws IOException {
-        final ResetFeatureStateRequest req = new ResetFeatureStateRequest();
-
+        final var req = new ResetFeatureStateRequest(RestUtils.getMasterNodeTimeout(request));
```

Comment:
This API param was missing - added here and in the JSON spec.

Reply:
Ah I didn't even realise this had docs. It's not really intended for end-users:

https://github.com/elastic/elasticsearch/blob/5e816686c5d2659a4ebf779dc69e0c5b1f91800b/docs/reference/features/apis/reset-features-api.asciidoc#L11